docs/accordproject-faq.md

```diff
@@ -0,0 +1,56 @@
+
+The result is a contract that is:
+
+
+* Searchable —
+* Analyzable —
+* Real-time —
+* Integrated —
+
+Consequently, contracts are transformed from business liabilities in constant need of management to assets capable of providing real business intelligence and value.
+
+### What is the Accord Project and what is its purpose?
+
+The Accord Project is a non-profit, member-driven organization that builds open source code and documentation for smart legal contracts for use by transactional attorneys, business and finance professionals, and other contract users. Open source means that anyone can use and contribute to the code and documentation and use it in their own software applications and systems free of charge.
+
+The purpose of the Accord Project is to establish and maintain a common and consistent legal and technical foundation for smart legal contracts. The Accord Project is organized into working groups focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution. The transactional working groups are assisted by the Technology Working Group, which builds the underlying open source code and specifications to codify the knowledge of the transactional working groups. More details about the internal governance of the Accord Project are available here.
+
+
+### How can I get involved?
+
+The working groups have calls at least once a month and use the Accord Project’s Slack group chat application (join by clicking here) for discussion. The dates, dial-in instructions, and agendas for the working groups are all listed in the Project’s public calendar and typically also in working group’s respective slack channels.
+
+A primary purpose of the working groups is to develop a universally accessible and widely used open source library of modular, smart legal contract and smart clause templates and models that reflect input from transactional attorneys and other experts. Smart legal contract templates are built according to the Project’s Cicero Specification.
+
+Members are expected to at least provide some feedback into the templates and models relevant to a particular working group. You can immediately get started contributing smart legal contract templates and models by using the Accord Project’s Template Studio.
+
+Another purpose of the working groups is to use and help develop an easy-to-use programming language for building and executing smart legal contracts called Ergo. The goals of Ergo are to be accessible and usable by non-technical professionals, portable across, and compatible with, a variety of environments such as SaaS platforms and different blockchains, and meeting security, safety, and other requirements. You can use the Accord Project’s Template Studio to create and test your smart legal contract models and run them on other software systems.
```

Comment:
Could provide a link for `... called Ergo.`

Unsure why but the second sentence seems awkward.

Last sentence could also include the VSCode Plugin in addition to Template Studio?

Reply:
`You can immediately start contributing smart legal contract templates and models by using the Accord Project Template Studio.` @irmerk?

Reply:
done